src/components/Events/Event.jsx

```diff
@@ -1,6 +1,22 @@
-import React from 'react';
-
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 const Event = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    const getEvent = async () => {
+      try {
+        const res = await axios.get(
```

Comment:
again
baseurl in a config file and the every other should refer that

Reply:
Will fix in next PR